### PR TITLE
✨(funmooc) add labels categories as search filter

### PR DESCRIPTION
**Pull request by jbpenrath**
## Purpose
A new category has been created on funmooc : "types" containing sub-categories espacially to know if a course is certifying or not.
C&P asks us to add this new category in search filters.

## Proposal
- [x] Add new search filter entry "Types"

Capture d’écran 2021-03-05 à 17 00 17

### CODE REVIEW

**Comment by sampaccoud** [comment]
I'm not sure about the term "label". How do certificates fall under this category?

**Comment by jbpenrath** [comment]
> I'm not sure about the term "label". How do certificates fall under this category?

This is a category that @ben created on fun-mooc. A discussion is open to improve these category names. (Labels & "Certificat")

File: sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -21,51 +21,71 @@ msgstr ""
 msgid "Unsorted uploads are not allowed."
 msgstr "Les téléchargements non classés sont interdits"
 
-#: funmooc/settings.py:219
+#: funmooc/settings.py:220
+msgid "Dashboard"
+msgstr ""
```
Comment (sampaccoud) [inline sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po:26]: translate all strings in this file?

File: sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -21,51 +21,71 @@ msgstr ""
...
+msgid "{base_url:s}/dashboard"
+msgstr ""
+
+#: funmooc/settings.py:224
 msgid "Profile"
 msgstr "Profil"
 
-#: funmooc/settings.py:220
+#: funmooc/settings.py:225
+msgid "{base_url:s}/u/(username)"
```
Comment (sampaccoud) [inline sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po:37]: I'm skeptical about the (username) pattern?

**Review by sampaccoud — APPROVED** [review]

File: sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (jbpenrath) [reply sites/funmooc/src/backend/locale/fr/LC_MESSAGES/django.po:26]: Translated in PR #202

**Pull request merged**